### ISSUE TITLE
Bug/Feature Request: Configure icon position in top panel, moves around currently

### ISSUE THREAD

**Issue by thyttan** [arononak/github-actions-gnome-extension#3]
Hi, thanks for the extension!

Similarly to how the extension "Just Perfection" allows me to choose "Clock Menu Position Offset" I'd like this option for this extension as well.

![image](https://github.com/arononak/github-actions-gnome-extension/assets/97237430/47bf08db-79db-4b42-bf5f-c2c1ccb6c604)

This is the visual I'm aiming for:
![image](https://github.com/arononak/github-actions-gnome-extension/assets/97237430/073bfed9-9828-43f7-93f3-a6c936098c11)

But this is how it looks when I log in to the desktop regardless of my efforts with other extensions:
![image](https://github.com/arononak/github-actions-gnome-extension/assets/97237430/b706c8f5-8dae-49e0-8a84-c7a74a6a78a3)

**Comment by arononak** [comment]
Great idea,

I thought about it but forgot to add it to the TODO list,

Added with high priority to the TODO list

Thanks !

**Comment by thyttan** [comment]
Thanks for implementing this!

Curiously my system bogged down completely when I did the change. Mouse started lagging util everything stood still - I even had a hard time powering the laptop off! But now that I rebooted everything seems fine and the icon is in the position I assigned it - I'm happy :)

![image](https://github.com/arononak/github-actions-gnome-extension/assets/97237430/b7575d4c-b0fe-44cc-b0e6-bf4fc4bcfdb0)

Don't know if it was some interference with other extensions. Both 'Just Perfection' and 'Unite' which I have installed manipulate positioning of stuff in the top panel.
